surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+  - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+  - big tech
+  - company
+  - startup
+  - university
+  - individual
+  - other
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+  - Direct end users: Users who run tools themselves
+  - Application operators: Users who interacts with existing running applications
```

Comment:
The more I am reading the two, the more it seems to be the same thing to me, maybe we should merge it?

Reply:
```suggestion
  - Direct end users: Users who run tools themselves
  - Application operators: Service and infrastructure providers
```